Makefile

```diff
@@ -1,7 +1,7 @@
 # Environment detection.
 
 UNAME := $(shell uname)
-CAIRO_2_VERSION = 2.10.0-rc.1
+CAIRO_2_VERSION = 2.10.0
 SCARB_VERSION = 2.9.2
```

Comment:
can you update scarb to 2.10.1 and alexandria test too?

https://github.com/keep-starknet-strange/alexandria/commit/74cafd429eef374730f2c6fed9a74753103c6fdb